doc/man1/openssl-pkeyutl.pod.in

```diff
@@ -242,8 +272,8 @@ engine I<id> for crypto operations.
 The operations and options supported vary according to the key algorithm
 and its implementation. The OpenSSL operations and options are indicated below.
 
-Unless otherwise mentioned all algorithms support the B<digest:>I<alg> option
-which specifies the digest in use for sign, verify and verifyrecover operations.
+Unless otherwise mentioned, all algorithms support the B<digest:>I<alg> option,
```

Comment:
It may be entirely clear to all readers that this is about `-pkeyopt` (rather than `-digest`). Is that worth addressing?